src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class RightTriangle implements Figure {
+    private static final int RIGHT_TRIANGLE_NUMBER = 2;
+    private int sid;
+    private String color;
+    public RightTriangle() {
+        this.sid = sid;
+        this.color = color;
+
+    }
+
+    @Override
+    public double getArea() {
+        return sid / RIGHT_TRIANGLE_NUMBER * sid;
+    }
```

Comment:
The class 'RightTriangle' should also implement the 'draw()' method, as required by the 'Figure' interface that it implements.